libs/nestjs-admin/src/tests/admin.module.spec.ts

```diff
@@ -16,7 +16,7 @@ describe('AppController', () => {
 
   it('should expose a default admin module ready to be used', async () => {
     const module: TestingModule = await Test.createTestingModule({
-      imports: [TypeOrmModule.forRoot(), DefaultAdminModule],
+      imports: [TypeOrmModule.forRoot(), CoreAdminModule],
```

Comment:
This test doesn't make sense anymore, but we need to test that the defaultAdmin works